src/util/htmlToJson.ts

```diff
@@ -200,9 +200,35 @@ const himalayaToBlock = (nodes: HimalayaNode[]): Block | Block[] => {
   return result
 }
 
+/** Gets all unique tag lists that is included in the text that we import. */
+const allTagsFinder = (nodes: Element[]): string[] => {
+  const tagLists = nodes.reduce((accum, current) => {
+    if (current?.children?.length) return [...accum, current.tagName, ...allTagsFinder(current.children as Element[])]
+    return current.tagName ? [...accum, current.tagName] : [...accum]
+  }, [] as string[])
+  return _.uniq(tagLists)
+}
+
+const tagsThatIsNotToBeStripped = [...allowedFormattingTags, 'span', 'li', 'ul']
+
+/** Generates dynamic regex expression. */
+const regexGenerator = (tags: string[]): RegExp => {
```

Comment:
`regexGenerator` is too generic a name for what this function does. It specifically generates a regex to remove tags, so I would call it `generateRegexToMatchTags` or `regexOfTags`.